src/views/InteractiveMap/CardDialogDesign.vue

```diff
@@ -378,9 +378,15 @@ export default Vue.extend({
           const fullMetabolite = addReaction.mnxReaction.metabolites.find(
             ({ mnx_id }) => mnx_id === m.id
           );
-          if (!fullMetabolite) return m;
-          if (!fullMetabolite.annotation) return m;
-          if (!fullMetabolite.annotation.bigg) return m;
```

Comment:
Oh, I see you didn't use this style elsewhere in code. How would most of you create reactionWithModelMetabolites? (you probably have a better way than reformatted version of my short guard clauses)

Reply:
Hmm, maybe there is better way I don't see - but I think the reformatted version, although also more verbose, at least is more readable, and I always value readability over conciseness.